diff --git a/clang-version-inc.py b/clang-version-inc.py
index eeffb625cc..0f7725a256 100755
--- a/clang-version-inc.py
+++ b/clang-version-inc.py
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
+
 import os
 import sys
 
@@ -8,7 +10,7 @@ import version
 version_string = '%s.%s.%s' % (version.major, version.minor, version.patch)
 
 if len(sys.argv) < 2:
-    print sys.stderr, "Usage: %s <Version.inc.in>"
+    print("Usage: %s <Version.inc.in>" % sys.argv[0], file=sys.stderr)
     sys.exit(1)
 
 with open(sys.argv[1], 'r') as f:
@@ -22,6 +24,6 @@ with open(sys.argv[1], 'r') as f:
 
     for line in f:
         line = line.strip()
-        for replace, to in replacements.iteritems():
+        for replace, to in replacements.items():
             line = line.replace(replace, to)
-        print line
+        print(line)
